app/controllers/opportunities_controller.rb

```diff
@@ -164,18 +164,18 @@ def geocode_location(location)
     Rails.cache.fetch("geocoded_location_of_#{location}") { User.where('LOWER(city) = ?', location.downcase).map { |u| [u.lat, u.lng] }.first || [0.0, 0.0] }
   end
 
-  def get_jobs_for(chosen_location, tag, page)
+  def get_jobs_for(chosen_location, tag, page, q = nil, remote_allowed = false, fulltime = false)
```

Comment:
could we use query instead ? The code should be clear. The param can stay as it is.